src/routes/Root.tsx

```diff
@@ -1,10 +1,109 @@
-import { Form } from "@lib/main";
+import { Button } from "@lib/gds-components";
 
-// The root will eventually be the starting page of the form (create or update)
 export const Root = () => {
   return (
-    <>
-      <Form />
-    </>
+    <div className="govuk-width-container app-width-container">
+      <main className="govuk-main-wrapper">
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-third">
+            <nav className="app-subnav" aria-labelledby="app-subnav-heading">
+              <ul className="app-subnav__section">
+                <li className="app-subnav__section-item ">
+                  Create or update a Local Plan timetable
+                </li>
+                <li className="app-subnav__section-item ">
+                  <a
+                    className="app-subnav__link govuk-link  govuk-link--no-underline"
+                    href="#"
+                    aria-current="page"
+                  >
+                    Create or update a Local Plan timetable
+                  </a>
```

Comment:
We don't have the URLs yet but since we're using `react-router`, should these be using `Link` elements instead?